soroban-sdk/src/env.rs

```diff
@@ -244,6 +244,28 @@ impl Env {
     pub fn log_value<V: IntoVal<Env, RawVal>>(&self, v: V) {
         internal::Env::log_value(self, v.into_val(self));
     }
+
+    #[doc(hidden)]
+    pub fn get_ledger_version(&self) -> u32 {
+        internal::Env::get_ledger_version(self).try_into().unwrap()
+    }
+
+    #[doc(hidden)]
+    pub fn get_ledger_sequence(&self) -> u32 {
+        internal::Env::get_ledger_sequence(self).try_into().unwrap()
+    }
+
+    #[doc(hidden)]
+    pub fn get_ledger_timestamp(&self) -> u64 {
+        let obj = internal::Env::get_ledger_timestamp(self);
+        internal::Env::obj_to_u64(self, obj)
+    }
+
+    #[doc(hidden)]
+    pub fn get_ledger_network_id(&self) -> Bytes {
+        let bin_obj = internal::Env::get_ledger_network_id(self);
+        bin_obj.in_env(self).try_into().unwrap()
```

Comment:
The `.try_into()` introduces a branch that will never be hit because we can trust that the host will always give us a `Bytes`. Can this line be replaced with the following?
```rust
unsafe { Bytes::unchecked_new(bin_obj) }
```
This is the same for the two functions above that return `u32`.